cernopendata/modules/fixtures/data/records/cms-tools-nanoaod-outreach-dimuon-spectrum.json

```diff
@@ -0,0 +1,143 @@
+    },
+    "accelerator": "CERN-LHC",
+    "authors": [
+      {
+        "name": "Wunsch, Stefan"
+      }
+    ],
+    "doi": "TODO: DOI_1",
+    "collections": [
+      "CMS-Tools"
+    ],
+    "date_published": "2019",
+    "experiment": "CMS",
+    "files": [
+      {
+        "checksum": "adler32:33f87981",
+        "size": 8098,
+        "uri": "https://raw.githubusercontent.com/cms-opendata-analyses/DimuonSpectrumNanoAODOutreachAnalysis/2012/dimuonSpectrum.ipynb"
+      },
+      {
+        "checksum": "adler32:c6ab092d",
+        "size": 2415,
+        "uri": "https://raw.githubusercontent.com/cms-opendata-analyses/DimuonSpectrumNanoAODOutreachAnalysis/2012/dimuonSpectrum.py"
+      },
+      {
+        "checksum": "adler32:51f0615b",
+        "size": 2839,
+        "uri": "https://raw.githubusercontent.com/cms-opendata-analyses/DimuonSpectrumNanoAODOutreachAnalysis/2012/dimuonSpectrum.C"
+      }
+    ],
+    "license": {
+      "attribution": "GNU General Public License (GPL) version 3"
+    },
+    "relations": [
+      {
+        "description": "",
+        "recid": "12341",
+        "type": "isRelatedTo"
+      },
+      {
+        "description": "",
+        "recid": "6004",
+        "type": "isRelatedTo"
+      },
+      {
+        "description": "",
+        "recid": "6030",
+        "type": "isRelatedTo"
+      },
+      {
+        "description": "",
+        "recid": "252",
+        "type": "isRelatedTo"
+      }
+    ],
+    "run_period": [
+      "Run2012A",
+      "Run2012B"
+    ],
+    "system_details": {
+      "description": "This code can be used with the CMS Open Data VM environment.",
+      "release": "ROOT 6.16 or later"
+    },
+    "type": {
+      "primary": "Software",
+      "secondary": [
+        "Tool"
```

Comment:
We can use "Analysis" here. I'll amend together with links to point to files on EOS instead of GitHub.